apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -94,7 +94,7 @@ export default class ProgressBubble extends React.Component {
       level.isLocked,
       level.isUnplugged,
       level.bonus,
-      false,
+      level.paired,
```

Comment:
This fixes the issue where pairing icons weren't being shown for bubbles in the teacher panel.  Bubbles in the header and unit overview page are still not showing the pairing icon and will be fixed separated (LP-2003).